recipes/pds4_tools/meta.yaml

```diff
@@ -0,0 +1,58 @@
+{% set name = "pds4_tools" %}
+{% set version = "1.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/pds4_tools-{{ version }}.zip
+  sha256: 8e6fdd0a9c4063c9a337b6ed3c52686da4f96bfba2a30e42ec57a6cf25fff459
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - python >=2.6,>=3.3
+  run:
+    - numpy
+    - python >=2.6,>=3.3
+    - tk
+
+test:
+  imports:
+    - pds4_tools
+  requires:
+    - pytest
+    - pip
+    - xorg-libx11
+  source_files:
+    - pds4_tools
+  commands:
+    - pip check
+    - pytest pds4_tools/tests/test_core.py
```

Comment:
```suggestion
```
This is a noarch python package. Full correctness tests are the responsibility of upstream and not needed here. Tests in the conda recipe should show that the packaging/dependencies/compilation was successful. Import tests and pip check are sufficient because there is no compilation.